task/power/src/main.rs

```diff
@@ -349,6 +359,45 @@ macro_rules! max5970_controller {
     };
 }
 
+fn trace_max5970(dev: &Max5970, sensor: SensorId) {
+    if let Ok(Volts(volts)) = dev.max_vout() {
+        use drv_i2c_devices::max5970::Register;
+
+        if volts < 12.0 {
+            return;
+        }
+
+        ringbuf_entry!(Trace::Max5970 {
```

Comment:
Maybe include `status1` here too?  In the implementations of interest, none of the bits can change, and `PROT` and `MODE` are sampled only at reset.  But `ONx` is a dynamic input and all these bits are probably as interesting as those in `status3`.